modules/apps/portal-vulcan/portal-vulcan-impl/src/main/java/com/liferay/portal/vulcan/internal/template/servlet/RESTClientHttpRequest.java

```diff
@@ -277,7 +279,7 @@ public Collection<Part> getParts() throws IOException, ServletException {
 
 	@Override
 	public String getPathInfo() {
-		return _httpServletRequest.getPathInfo();
+		return _path;
```

Comment:
I guess for the previous case (Tomcat) _path and pathInfo contained the same info, right?